Forms/AHKForm.cs

```diff
@@ -106,9 +112,25 @@ private void InitializeCheckAsThreeState()
                 }
         }
 
-        private void noclicksample_CheckedChanged(object sender, EventArgs e)
+        private void SetLegendDefaultValues()
         {
+            this.cbWithNoClick.ThreeState = true;
+            this.cbWithNoClick.CheckState = System.Windows.Forms.CheckState.Indeterminate;
+            this.cbWithNoClick.AutoCheck = false;
+            this.cbWithClick.CheckState = System.Windows.Forms.CheckState.Checked;
+            this.cbWithClick.ThreeState = true;
+            this.cbWithClick.AutoCheck = false;
+        }
```

Comment:
This is necessary to pre-setup the legend configuration and disallow users changing it.